### PR TITLE
Updating docs for preview release

### PR DESCRIPTION
**Pull request by pjlittle**
- Added docs for preview release, including current screenshots
- Added MIT license back
- Separated out `CONTRIBUTING.md`
- added `.nvmrc` for convenience (will use that version of node, if you have nvm locally)

Easier to view the markdown directly on GH: https://github.com/Microsoft/VoTT/tree/pelittle/preview-docs

**Please help me check for typos and/or anything incorrect!**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,63 +2,131 @@
...
+## Labeling a Video
+
+Coming soon!
+
+## Exporting Labels
+
+Once assets have been labeled, they can be exported into a variety of formats:
+
+* [Azure Custom Vision Service](https://azure.microsoft.com/en-us/services/cognitive-services/custom-vision-service/)
+* TensorFlow (Pascal VOC and TF records)
```
Comment (JacopoMangiavacchi) [inline README.md]: nit: TFRecords

File: README.md [hunk 2]
```diff
@@ -2,63 +2,131 @@
...
+
+# Using VoTT
+
+## Creating Connections
+
+VoTT is a 'Bring Your Own Data' (BYOD) application. In VoTT, connections are used to configure source (assets to labeled) and target (where labels should be exported to) data sources.
+
+Connections can be setup and shared across projects. Connections have been designed using an extensible provider model -- new source/target provides can easily be added.  Currently, VOTT supports:
+
+* Azure Blog Storage
```
Comment (pjlittle) [inline README.md]: whoops... blog --> blob

File: README.md [hunk 3]
```diff
@@ -2,63 +2,131 @@
...
+* The ability to label images or video frames
+* Extensible model for importing data from local or cloud storage providers
+* Extensible model for exporting labeled data to local or cloud storage providers
 
-### Issues
+VoTT helps facilitate an end-to-end machine learning pipeline:
 
-- Look for duplicate issues & comment on thread if experiencing something similar
-- Fill in template information (platform, OS, version, screenshots, etc.)
+![alt text](docs/images/ml-workflow.png "Machine Learning Workflow")
```
[the remark this answers is not in the document]
Comment (pjlittle) [reply README.md]: can you show me what you mean? I can go either way on this...

File: README.md [hunk 4]
```diff
@@ -2,63 +2,131 @@
...
+
+![alt text](docs/images/new-project.png "New Project")
+
+### Project Settings
+
+Project settings can be modified after a project has been created, by clicking on the 'Project Setting' icon in the left hand navigation bar.
+
+#### Security Token
+
+Some project settings can include sensitive values, such as API keys or other shared secrets. Each project has will generate a security token that can be used to encrypt/decrypt sensitive project settings.
```
[the remark this answers is not in the document]
Comment (pjlittle) [reply README.md]: I waffled on mentioning shared projects, in lieu of breaching the topic of collaboration, but makes sense, given that you securing sensitive values in a shared setting would be a common scenario.

**Comment by pjlittle** [comment]
I'm going to add a TOC as well.

File: README.md (hunk 1 above)
Comment (pjlittle) [reply README.md]: done!

File: README.md [hunk 5]
```diff
@@ -2,63 +2,131 @@
...
+
+* Azure Blog Storage
+* Bing Image Search
+* Local File System
+
+To create a new connection, click the 'New Connection' icon, in the left hand navigation bar:
+
+![alt text](docs/images/new-connection.png "New Connection")
+
+## Creating a New Project
```
[the remark this answers is not in the document]
Comment (pjlittle) [reply README.md]: updated.

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (pjlittle) [reply README.md]: done. also added one more screenshot.

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (pjlittle) [reply README.md]: leaving as-is.

**Pull request merged**